### PR TITLE
Milestone: Complete Milestone 5 and 6

### PR DESCRIPTION
**Pull request by oxenprogrammer**
# Project 2: Building with Responsive Design

## Technology Used
- HTML5
- CSS3 (Flexbox, CSS3 Grid, and Floats, Media Queries)

### Description
- This is the clone for the site www.thenextweb.com
- Designed with responsiveness for desktop, tablet and mobile

### Project link 
   https://oxenprogrammer.github.io/thenextweb-clone/

### Desktop version
Desktop

### Tablet version
Tablet

### Mobile version
Mobile

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
-# thenextweb-clone
\ No newline at end of file
+# Project 2: Building with Responsive Design
+
+![screenshot](./Desktop.jpg)
+
+![screenshot](./Mobile.jpg)
```
Comment (arslanbisharat) [inline README.md]: - [x] Please add the proper description of the image you are presenting in Readme. You also need to add a Tablet version image here. When you make a tablet version of the webpage, Please add its screenshot here.

File: README.md [hunk 2]
```diff
@@ -1 +1,68 @@
...
+## Built With
+
+- HTML5
+- CSS3#
+
+
+## Live Demo
+
+[Live Demo Link](https://oxenprogrammer.github.io/thenextweb-clone/)
+
```
Comment (arslanbisharat) [inline README.md:18]: - [x] This live link is outdated. Please add a new link or update the link

File: README.md [hunk 3]
```diff
@@ -1 +1,68 @@
...
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+N/A
```
Comment (arslanbisharat) [inline README.md]: Remove this part

File: README.md [hunk 4]
```diff
@@ -1 +1,68 @@
...
+
+### Setup
+Clone or Download the project from github.com
```
Comment (arslanbisharat) [inline README.md]: - [x] Can you please tell me how to clone it?

File: README.md [hunk 5]
```diff
@@ -1 +1,68 @@
...
+### Install
+No installation required, simply right click on the index.html file and run it on your favorite browser.
+
+### Deployment
+Deploy on gh pages for testing purposes
```
Comment (arslanbisharat) [inline README.md:33]: Remove this part too.

File: README.md [hunk 6]
```diff
@@ -1 +1,68 @@
...
+
+- GitHub: [@githubhandle](https://github.com/oxenprogrammer)
+- Twitter: [@twitterhandle](https://twitter.com/ox_emmy)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/emanuel-okello-1217b4b3/)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/aliciapaz)
+- Twitter: [@twitterhandle](https://twitter.com/aliciap38180090 )
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/alicia-rojas-71468418a/ )
```
Comment (arslanbisharat) [inline README.md]: Add authors name, please

File: README.md [hunk 7]
```diff
@@ -1 +1,68 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- https://www.theodinproject.com/
+- https://www.thenextweb.com
+
+## 📝 License
+
+This project is [MIT](./LICENSE.txt) licensed.
```
Comment (arslanbisharat) [inline README.md]: Please add the proper license link here.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_

This is a great opportunity to learn. Microverse is here to help you. 

**Please have a look at the suggestions to get more understanding.**

- [x] Your project is looking amazing but Your readme is outdated. Please look at the suggestions I provided below. 

### Desktop

- [x] Everything looks fine here but the sticky navbar is missing please work on it. 

- [x] Please move this section to here and add | after each item. 
chrome-capture (18)


### Medium

- [x] Your whole webpage missing medium styling. You are mistakenly are given small version styling to the medium screen too.  Here are  some suggestions:

- On the navbar TWN logo appears at the leftmost part
- after the navbar in 3 article section TWN logo disappears
- Latest News section aligns 3 articles per row
- Latest funding section aligns 3 articles per row
- Categories (Apps, Gear, Tech, etc) section aligns 2 category boxes per row
- Latest deal section aligns 3 articles per row

### Small

On a small screen, this section should have rows for each item. 
**Clone**
chrome-capture (19)
**Original**
chrome-capture (20)

- [x] Text and image are not rendering properly but overflowing on all the latest additions and categories section. Please have a look there. 
chrome-capture (21)

- [x] [Optional] Please add a small description of your project in the about section of your rope. 


**Please make the changes and request a re-review for the given project**

If you get stuck somewhere, don't hesitate to reach me out or schedule s 1:1 meeting with a Full-time TSE. 

Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by Srjnnnn — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 
Hello team! It is simply a masterpiece. :muscle:  I really like your page and also thanks for making the requested changes. There is just one minor issue before moving on.

- [x] On your footer there is a broken image. Kindly fix this.
footer
### Optional:

- [x] I think your Readme contains too many images. Put one or two. Too many images make your readme not-readable I think.

Again, it is a great job. Keep up the good work. If you have any questions about the review, feel free to reach me via Slack.
_Eyüp Sercan UYGUR_

**Review by jubaan — APPROVED** [review]
#
# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

#### :clap: :clap: Great job :hammer_and_pick:!! There are no more issues left to be addressed. 

COMMENT:

- Change the name of the social handles to the actual user name, this would be more awesome, than the plain @twitterhandle. Example [@jubaan](https://www.github.com/jubaan).

## See you at the top :rocket:!

# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

**Pull request merged**